src/routes/components/TextField.tsx

```diff
@@ -0,0 +1,571 @@
+                  <GoADropdownItem label="Prince Edward Island" value="prince-edward-island" />
+                  <GoADropdownItem label="Quebec" value="quebec" />
+                  <GoADropdownItem label="Saskatchewan" value="saskatchewan" />
+                </GoADropdown>
+              </GoAFormItem>
+
+              <GoAFormItem label="Postal Code">
+                <GoAInput name="postalCode" type="text" value="" onChange={noop} width="100%" />
+              </GoAFormItem>
+            </GoABlock>
+          </Sandbox>
+
+          <h3 id="ask-user-for-birthday">Ask a user for their birthday</h3>
+          <Sandbox flags={["reactive"]}>
+            <GoAFormItem
+              label="When is your birthday?"
+              labelSize="large"
+              helpText={"For example, 27 11 2007"}>
+              <GoABlock gap="m" direction="row">
+                <GoAFormItem label="Day">
+                  <GoAInput
+                    onChange={noop}
+                    value=""
+                    name="bandNo"
+                    type="number"
+                    width="60px"
+                    min={1}
+                    max={31}
+                  />
+                </GoAFormItem>
+                <GoAFormItem label="Month">
+                  <GoAInput
+                    onChange={noop}
+                    value=""
+                    name="family"
+                    type="number"
+                    width="60px"
+                    min={1}
+                    max={12}
+                  />
+                </GoAFormItem>
+                <GoAFormItem label="Year">
+                  <GoAInput
+                    onChange={noop}
+                    value=""
+                    name="position"
+                    type="number"
+                    width="88px"
+                    min={1900}
+                    max={2023}
+                  />
+                </GoAFormItem>
+              </GoABlock>
+            </GoAFormItem>
+          </Sandbox>
+
+          <h3 id="search">Search</h3>
+          <Sandbox flags={["reactive"]}>
+            <GoABlock gap="m" direction="row">
+              <GoAInput type="search" name="search" value="" onChange={noop} leadingIcon="search" />
+              <GoAButton type="primary" onClick={noop}>
+                Search
+              </GoAButton>
+            </GoABlock>
+          </Sandbox>
+
+          <h3 id="phone">Phone number</h3>
+          <Sandbox flags={["reactive"]}>
+            <GoAFormItem label="Phone number">
+              <GoAInput onChange={noop} value="" name="phone" leadingContent={<span>+1</span>} />
+            </GoAFormItem>
+          </Sandbox>
+
+          <h3 id="ask-for-costs">Ask a user for dollar amounts or costs</h3>
+          <Sandbox flags={["reactive"]}>
+            <GoAFormItem label="Tuition">
+              <GoAInput onChange={noop} value="" name="tuition" leadingContent={<span>$</span>} />
```

Comment:
You shouldn't need to wrap the leadingContent with a `span` tag, just a string works. It doesn't matter now, but if we are able to get the slots rendering in the example code it will be unnecessary code.